assets/dev/js/editor/components/template-library/views/parts/templates.js

```diff
@@ -339,6 +341,22 @@ const TemplateLibraryCollectionView = Marionette.CompositeView.extend( {
 		this.removeScrollListener = () => scrollableContainer.off( 'scroll', listener );
 	},
 
+	onCreateNewFolderClick() {
+		const activeSource = elementor.templates.getFilter( 'source' );
+
+		if ( 'cloud' !== activeSource ) {
+			return;
+		}
+
+		elementor.templates.createFolder( {
+			source: activeSource,
+		},
+		{
+			onSuccess: () => {
+				elementor.templates.showTemplates();
```

Comment:
I think we have a problem with `elementor.templates.showTemplates();`, the Tabs become unresponsive after it.s been called. It looks like this is the issue: https://github.com/elementor/elementor/blob/main/assets/dev/js/editor/components/template-library/manager.js#L598 - everything still works fine if I delete this line `self.layout.setHeaderDefaultParts()`;, but once again it's a BC concern that we might not know if we remove it what it will break.